recipe/0002-Don-t-link-to-fmt-library.-We-use-spdlog-s-embedded-.patch

```diff
@@ -1,32 +1,44 @@
-From f9733cf7f72f3997566679f37641a3240392e05f Mon Sep 17 00:00:00 2001
-From: Jean-Christophe Morin <jcmorin@anaconda.com>
```

Comment:
You should keep the header. You can put your name if you want, but we should keep the header to know where it's coming from.

Reply:
I think this patch was regenerated by Ian using `git diff`, thus no header.
We are not using it anymore tho, and I don't know what commit he used for it.

Reply:
This patch is unused now, I'll keep the change for now in case Ian was going to use it for something smart